src/message/outgoing-message.js

```diff
@@ -0,0 +1,85 @@
+const Transform = require('readable-stream').Transform;
+
+const { Packet, HEADER_LENGTH, TYPE } = require('../packet');
+
+const BufferList = require('bl');
+
+function isValidType(type) {
+  switch (type) {
```

Comment:
I think this function should go into Packet.js. And a corresponding unit test would be nice.